spec/acceptance/unbound_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'unbound class' do
+  describe 'running puppet code' do
+    it 'work with no errors' do
+      pp = 'class {\'::unbound\': }'
```

Comment:
might wanna just use `"class {'::unbound': }"`, and, since we're testing >=4, we can also drop the `::` and go with `"class {'unbound': }"`